gateway/modules/crud/operations.go

```diff
@@ -54,7 +58,21 @@ func (m *Module) Read(ctx context.Context, dbAlias, project, col string, req *mo
 		return nil, err
 	}
 
-	n, result, err := crud.Read(ctx, project, col, req)
+	var n int64
+	var result interface{}
+	var dataLoader *dataloader.Loader
+	var key model.ReadRequestKey
+	if req.Options != nil {
+		if req.Options.Prefix != "" {
+			key = model.ReadRequestKey{DBType: dbAlias, Col: col, HasOptions: req.Options.HasOptions, Req: *req}
+			dataLoader = newLoaderMap().get(fmt.Sprintf("%s-%s", req.Options.Prefix, uuid.New().String()), m)
```

Comment:
A few problems here
- If you are creating a newLoaderMap for each request you will never batch the read operations. Have a single globally defined loadermap
- If you add a unique uuid for every key here, you'll never batch the read operations, since each read operations will have a different data loader struct.